wp-graphql-seopress.php

```diff
@@ -32,18 +32,19 @@ function () {
 		register_graphql_object_type(
 			'SEOPress',
 			array(
+				'description' => __('The SEOPress schema data', 'wp-graphql'),
 				'fields' => array(
```

Comment:
:warning: **Warning**: Array double arrow not aligned correctly; expected 6 space(s) between &quot;'fields'&quot; and double arrow, but found 1 (*WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned*).